tests/cmux.py

```
@@ -211,11 +256,14 @@ def _default_socket_path() -> str:
 
     if bundle_id == _DEFAULT_DEBUG_BUNDLE_ID:
         tagged = glob.glob("/tmp/cmux-debug-*.sock")
```

Ruff (0.15.12)

[error] Probable insecure usage of temporary file or directory: "/tmp/cmux-debug-*.sock"

(S108)